CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 14.0.0 (xx/xx/23)
+
+### Breaking Changes
+
+Please familiarize yourself with the following potentially disruptive changes
+in Teleport 14 before upgrading.
+
+#### `tsh db` --db-user and --db-name defaults
```

Comment:
```suggestion
#### `tsh db --db-user` and `--db-name` defaults
```

Reply:
What is the intent of this suggestion? Would this be better / more clear:
`tsh [proxy] db [login | connect]` flags `--db-user` and `--db-name`

Reply:
@alexfornuto